src/maker/handlers.rs

```diff
@@ -7,7 +7,7 @@
 //! The file includes functions to validate and sign contract transactions, verify proof of funding, and handle unexpected recovery scenarios.
 //! Implements the core functionality for a Maker in a Bitcoin coinswap protocol.
 
-use std::{net::IpAddr, sync::Arc, time::Instant};
+use std::{net::IpAddr, sync::Arc, thread, time::Instant};
```

Comment:
Any specific reason for importing?
Since IMO, we are just using `thread` once here -> so would be better to use 
`std::thread::` here.

Reply:
It was changed due to your comment only.
Look at https://github.com/citadel-tech/coinswap/pull/324#discussion_r1888212049